pyvenv.cfg

```diff
@@ -0,0 +1,3 @@
+home = /Users/charleskramer/opt/anaconda3/bin
```

Comment:
looks like you missed one last file that should be gitignored/removed

Reply:
ah let me get that one now. fixed the f string thing too.